game-app/game-core/src/main/java/games/strategy/engine/data/DynamicallyModifiable.java

```diff
@@ -46,4 +45,9 @@ default MutableProperty<?> getPropertyOrThrow(final String name) {
     return getProperty(name)
         .orElseThrow(() -> new IllegalArgumentException("unknown property named '" + name + "'"));
   }
+
+  default void setPropertyOrThrow(String name, String value)
+      throws MutableProperty.InvalidValueException {
+    getPropertyOrThrow(name).setValue(value);
```

Comment:
This is only used by test code right? I think it would be better to move this helper method into test code

Reply:
Done.